src/ui/public/field_editor/components/scripting_call_outs/help_flyout.js

```diff
@@ -37,72 +39,157 @@ export const ScriptingHelpFlyout = ({
+                    <EuiIcon type="link" />
+                  </EuiLink>
+                )
+              }}
+            />
           </p>
           <p>
-            Kibana currently imposes one special limitation on the painless scripts you write. They cannot contain named functions.
+            <FormattedMessage
+              id="common.ui.fieldEditor.help.kibana.label"
+              // eslint-disable-next-line max-len
+              defaultMessage="Kibana currently imposes one special limitation on the painless scripts you write. They cannot contain named functions."
+            />
           </p>
           <p>
-            Coming from an older version of Kibana? The {(
-              <EuiLink
-                target="_window"
-                href={getDocLink('scriptedFields.luceneExpressions')}
-              >
-                Lucene Expressions <EuiIcon type="link"/>
-              </EuiLink>
-            )} you know and love are still available. Lucene expressions are a lot like JavaScript,
-              but limited to basic arithmetic, bitwise and comparison operations.
+            <FormattedMessage
+              id="common.ui.fieldEditor.help.lucene.common.label.detail"
+              // eslint-disable-next-line max-len
+              defaultMessage="Coming from an older version of Kibana? The {lucene} you know and love are still available. Lucene expressions are a lot like JavaScript, but limited to basic arithmetic, bitwise and comparison operations."
+              values={{
+                lucene: (
+                  <EuiLink
+                    target="_window"
+                    href={getDocLink('scriptedFields.luceneExpressions')}
+                  >
+                    <FormattedMessage id="common.ui.fieldEditor.help.lucene.common.label.lucene" defaultMessage="Lucene Expressions" />
+                    &nbsp;<EuiIcon type="link" />
+                  </EuiLink>
+                )
+              }}
+            />
           </p>
           <p>
-            There are a few limitations when using Lucene Expressions:
+            <FormattedMessage
+              id="common.ui.fieldEditor.help.lucene.limits.label"
+              defaultMessage="There are a few limitations when using Lucene Expressions:"
+            />
           </p>
           <ul>
-            <li> Only numeric, boolean, date, and geo_point fields may be accessed</li>
-            <li> Stored fields are not available</li>
-            <li> If a field is sparse (only some documents contain a value), documents missing the field will have a value of 0</li>
+            <li>
+              <FormattedMessage
+                id="common.ui.fieldEditor.help.lucene.limits.types.label"
+                defaultMessage="Only numeric, boolean, date, and geo_point fields may be accessed"
+              />
+
+            </li>
+            <li> <FormattedMessage
```

Comment:
Can you indent `<FormattedMessage>` on the next line similar to how you did on line 128? Same with the others in this file.